e2e_samples/parking_sensors_synapse/synapse/workspace/linkedService/Ls_KeyVault_01.json

```diff
@@ -1,11 +1,10 @@
 {
-	"name": "Ls_KeyVault_01",
-	"type": "Microsoft.Synapse/workspaces/linkedservices",
-	"properties": {
-		"type": "AzureKeyVault",
-		"annotations": [],
-		"typeProperties": {
-			"baseUrl": "https://mdwdops-kv-dev-dosyn1.vault.azure.net/"
-		}
-	}
+    "name": "Ls_KeyVault_01",
+    "properties": {
+        "annotations": [],
+        "type": "AzureKeyVault",
+        "typeProperties": {
+            "baseUrl": "https://mdwdops-kv-prod-lace21.vault.azure.net/"
```

Comment:
is this file being accidently checked in? Or do we want to provide more general/descriptive name just like others?

Reply:
Good question -- basically this file will get checked-in to the branch you are developing and testing on when you setup the git integration with the dev synapse workspace. I'd have to think of an easy, non-error prone way so that we could potentially just blow-away any developer-specific changes involved in testing the solution (ei. pointing baseUrl to my own KeyVault) in the workspace folder before raising a PR.

For now, I'll clean up the baseUrl manually :)

Reply:
Just cleaned it up :)